create_derivatives/routines.py

```diff
@@ -78,8 +79,30 @@ class ManifestMaker:
 
 
 class AWSUpload:
-    # TO DO: upload files and PDFs
-    pass
+
+    def __init__(self):
+        self.aws_client = AWSClient(*settings.AWS)
+
+    def run(self, replace):
+        uploaded_bags = []
+        for bag in Bag.objects.filter(process_status=Bag.MANIFESTS_CREATED):
+            pdf_dir = join(bag.bag_path, 'data', 'PDF')
+            jp2_dir = join(bag.bag_path, 'data', 'JP2')
+            manifest_dir = join(bag.bag_path, 'data', 'MANIFEST')
+            check_dir_exists(pdf_dir)
+            check_dir_exists(jp2_dir)
+            check_dir_exists(manifest_dir)
+            for src_dir, target_dir, file_type in [
+                (pdf_dir, "pdfs", "PDF file"),
+                (jp2_dir, "images", "JPEG2000 files"),
+                    (manifest_dir, "manifests", "Manifest file")]:
```

Comment:
Indentation is wonky here.

Reply:
That is what linting suggested for me. Would you suggested overriding it?

Reply:
I would suggest making sure you don't have a mix of spaces and tabs, and/or indenting lines 96/97 to match line 98